{
  boolean result=false;
  try {
    git.status().call();
  }
 catch (  Exception e) {
    result=true;
  }
  return result;
}
